### ISSUE TITLE
2.7.0 Thumbnail Generation Broken

### ISSUE THREAD

**Issue by kolbykruger** [modxcms/revolution#14172]
### Summary
Upgraded to 2.7.0, hover over any image in the file menu. I have tested this with new files and files that previously existed in this MODX build. The file types I have tested with are `.jpg` and `.png`.

I receive this error that logs for every file I hover over in the file browser:
```
[2018-11-30 09:40:52] (ERROR in modProcessor::run @ /home/***/***/core/model/modx/modprocessor.class.php : 177) Flat file processor support is deprecated since version 2.7.0.
 ```
I did read up [Mark Hamstra's 2.7.0 post](https://www.markhamstra.com/modx/2018/deprecated-notices-in-modx-2.7/?utm_source=rss), however this seems to be `modxProcessor` and not an installed extra causing the issue. I could be totally incorrect here...

### Screenshot for reference
![untitled-1](https://user-images.githubusercontent.com/10890005/49305954-c9a76100-f49e-11e8-9262-0ee8235d8376.png)

**Comment by JoshuaLuckers** [comment]
Thanks for taking time to report this issue to make MODX better.

The errror in the log is actually only a warning. 

Do thumbnails work if you delete the MODX and/or browser cache?

**Comment by kolbykruger** [comment]
Oddly enough, deleting the core/cache folder fixed thumbnails for the default filesystem, but not the media sources I created. 

I should clarify however that in the media browser cmp, the thumbnails work properly. This only happens in the left sidebar files tab for media sources I created.

**Comment by JoshuaLuckers** [comment]
Thanks for the extra information. I can confirm that the thumbnails are not working when using a custom media source. It looks like the `basePath` set for the Media Source is not including in the URL for the thumbnail.

**Comment by kolbykruger** [comment]
Good find. Thanks for investigating!

**Comment by JoshuaLuckers** [comment]
Actually, that's only the case if it's an SVG file.. other files are going trough the phpthumb processor and that does include the media source.

**Comment by jonleverrier** [comment]
Strange - I can't reproduce this issue locally or in a production environment running 2.7.0.

1. I created a custom media source
2. I hovered over the images (jpg format) in the file tab

Images displayed fine on hover, and did not generated the flat file error message (when I access the media manager in the top manager, this generated the error message). Personally, i'd forget about the error messages for the time being.

Could you post any more info about your environment? Or media source settings?

**Comment by JoshuaLuckers** [comment]
I was able to reproduce it.
When I hovered the images the thumbnail was not generated. If I call the URL for the thumbnail manually it did generate the thumbnail and refreshing the page showed the image on hover.

**Comment by kolbykruger** [comment]
Turns out it may have been a cache thing with the update... I am not entirely sure though. It is magically working today, without me changing anything. Could have been hanging onto something from before the update?

**Comment by Ibochkarev** [comment]
@kolbykruger  Hi! Still relevant?

**Comment by kolbykruger** [comment]
I believe it was resolved as of 2.7.0, after deleting the site cache, and giving it a few days.
